src/adapter/mysql.cr

```diff
@@ -5,7 +5,7 @@ require "mysql"
 class Granite::Adapter::Mysql < Granite::Adapter::Base
   # Using TRUNCATE instead of DELETE so the id column resets to 0
   def clear(table_name)
-    statement = "TRUNCATE #{table_name}"
+    statement = "TRUNCATE #{quote_table_name(table_name)}"
```

Comment:
This is part of where I was hoping to head with #122, thank you for implementing it.